src/webgpu/api/validation/capability_checks/features/texture_formats.spec.ts

```diff
@@ -161,15 +162,15 @@ g.test('canvas_configuration')
       usage: GPUTextureUsage.COPY_SRC | GPUTextureUsage.COPY_DST,
     };
 
-    if (enable_required_feature) {
-      t.expectValidationError(() => {
-        ctx.configure(canvasConf);
-      });
-    } else {
-      t.shouldThrow('TypeError', () => {
-        ctx.configure(canvasConf);
-      });
-    }
+    const expectedError =
+      enable_required_feature &&
+      (kCanvasTextureFormats as unknown as Array<GPUTextureFormat>).includes(format)
```

Comment:
IMHO this test can now be removed as we always get a `TypeError` if we are not in the supported canvas formats, so the test is redundant with the one modified in https://github.com/gpuweb/cts/pull/3997/files

Reply:
As I wrote in PR description: if we ever add supported context format that requires features this will be relevant, but I am not opposing deletion.

Reply:
Gotcha, ok sounds good.